boards/OAK-T.json

```diff
@@ -0,0 +1,14 @@
+{
+    "board_config":
+    {
+        "name": "OAK-T",
+        "revision": "R3M1E2",
+        "cameras":{
```

Comment:
CC: @CenekAlbl 

I assume we'll want to specify the thermal imager here as well @zrezke ?

Reply:
@zrezke could you please add the missing thermal camera?

Reply:
@ahrastnik added.

Reply:
I think we need an approve of @themarpe too here, because of the requested changes.